src/devices.js

```diff
@@ -530,7 +535,7 @@ function assembleInstallSteps(steps) {
                     "user:connection-lost",
                     step.resumable ? runStep : restartInstall
                   );
-                } else if (error.message.includes("Killed")) {
+                } else if (error.message.includes("killed")) {
```

Comment:
Maybe make case-insensitive?

Reply:
This looks for a handled standardized error in promise-android-tools, which is always lowercase now.